tonic/tonic.go

```diff
@@ -269,6 +270,12 @@ func Handler(f interface{}, retcode int, options ...func(*Route)) gin.HandlerFun
 				handleError(c, InputError(err.Error()))
 				return
 			}
+			// validating query and path inputs if they have a validate tag
+			validatorObj := validator.New(&validator.Config{TagName: "validate"})
```

Comment:
Why don't we just use `binding` as tag name here, so we don't have to deprecate it.
Also, this would mean that the validation would occur two times:
- by gin
- by our own validator instance
To resolve the issue, two possibilities:
- use `gin.DisableBindingValidation` globally in tonic
- implements the `gin.Binding` interface for JSON and other formats ourselves, and omit the validation in this implem.
cf @loopfz @Rbeuque74